[index] misc wording and styling improvements

jankeromnes opened the pull request:
I didn't touch the styling too much, but fixed typos and improved wordings where it made sense to me.

<details>
<summary>(old screenshot)</summary>
<img alt="Screenshot_2019-03-19 Gitpod" src="https://user-images.githubusercontent.com/599268/54599374-c7134c00-4a3a-11e9-9a72-7e64f36353b4.png">
</details>

svenefftinge commented on src/pages/index.tsx:
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
                     <div style={{ margin: '40px 0px' }}>
                         <UrlAnimation />
                     </div>
+                    <div style={{ marginTop: 30 }}>
```
For the first item switching text and image doesn't improve IMHO.

svenefftinge commented on src/pages/index.tsx:
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
                 contents = <div style={{ margin: 40 }}>
-                    <strong>Have a button on every GitHub page.</strong>
+                    <img src={clean(BrowserExtension)} />
+                    <strong>Have a button on every GitHub page</strong>
```
+1 Like this better than before

svenefftinge commented on src/pages/index.tsx:
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
             case 2: {
                 contents = <div style={{ margin: 40 }}>
-                    <p><strong>The GitHub app builds your projects</strong> so you don't have to when starting to code.</p>
+                    <img style={{ borderRadius: 2, margin: 0, width: 700 }} src={GitHubApp} />
```
Here the image is bad anway. We need to see how we can improve the message and better support it visually.

svenefftinge commented on src/pages/index.tsx:
```diff
@@ -55,46 +55,46 @@ export default class IndexPage extends React.Component<{}, IndexPageState> {
         }
         return <div style={{ height: 600 }}>
-            <h2 >How It Works:</h2>
+            <h2 >Get Disposable Workspaces</h2>
```
We haven't talked about 'Disposable Workspaces' so far. I think we should find something simpler.

jankeromnes commented:
- Rebased (new git-graph)
- Replaced "Get Disposable Workspaces" with "Get Started Coding"
- Trying with visuals between super-text and sub-text

![Screenshot_2019-03-19 Gitpod(2)](https://user-images.githubusercontent.com/599268/54618262-02c20c00-4a63-11e9-9670-0ececd008b43.png)
